platooning_control_ihp/src/platoon_control_ihp.cpp

```diff
@@ -49,25 +49,19 @@ namespace platoon_control_ihp
         pnh_->param<double>("lowpassGain", config.lowpassGain, config.lowpassGain);
         pnh_->param<double>("lookaheadRatio", config.lookaheadRatio, config.lookaheadRatio);
         pnh_->param<double>("minLookaheadDist", config.minLookaheadDist, config.minLookaheadDist);
-
-        // Global params (from vehicle config)
         pnh_->getParam("/vehicle_id", config.vehicleID);
         pnh_->getParam("/vehicle_wheel_base", config.wheelBase);
-        pnh_->getParam("/control_plugin_shutdown_timeout", config.shutdownTimeout);
-        pnh_->getParam("/control_plugin_ignore_initial_inputs", config.ignoreInitialInputs);
 
         pcw_.updateConfigParams(config);
         config_ = config;
 
 	  	// Trajectory Plan Subscriber
-        //   TODO: update trajectory executor to call this plugin
```

Comment:
Are we certain this TODO is no longer relevant (@sainaramyar )?  I don't see any mods to the trajectory executor here, so what is different for IHP?